### PR TITLE
Code coverage

### PR DESCRIPTION
**Pull request by juanfabrega**

### CODE REVIEW

File: jest.config.js [hunk 1]
```diff
@@ -13,4 +13,10 @@ module.exports = {
   },
   coverageDirectory: './coverage/',
   collectCoverage: true,
+  collectCoverageFrom: [
+    '**/*.{js,jsx}',
+    '!**/*.stories.{js,jsx}',
+    '!**/node_modules/**',
+    '!**/vendor/**',
```
Comment (nathanyoung) [inline jest.config.js:20]: I might be misunderstanding this, but shouldn't we not collect coverage from these things?

**Comment by juanfabrega** [comment]
The not (!) operator is there for that exact purpose. :)

File: jest.config.js (hunk 1 above)
Comment (juanfabrega) [reply jest.config.js:20]: @nathanyoung correct we are excluding these. (!)

Comment (nathanyoung) [reply jest.config.js:20]: yup, i was misunderstanding 👍 

do we need to add .mdx too?

Comment (juanfabrega) [reply jest.config.js:20]: @nathanyoung hmm, I think you're correct on that one. Let me confirm.

Comment (juanfabrega) [reply jest.config.js:20]: @nathanyoung it does not appear that mdx files get picked up by the jest runner at all actually. Which actually makes me wonder if they are getting linted, will confirm but that is a separate PR. This one is good to go.

**Pull request merged**